opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchScrollRequest.java

```diff
@@ -152,6 +153,6 @@ public void reset() {
    */
   @Override
   public String toCursor() {
-    return scrollId;
+    return needClean ? "" : scrollId;
```

Comment:
Does this change have any impact on the external behavior and protocol that JDBC needs to change accordingly?

Reply:
This has changed but no, it didn't affect external behaviour -- toCursor was only used for plan serialization for pagination.